packages/devtools_app/lib/src/utils.dart

```diff
@@ -71,6 +71,13 @@ String prettyPrintBytes(
   int mbFractionDigits = 1,
   bool includeUnit = false,
 }) {
+  if (bytes.abs() < 52) {
```

Comment:
Add a comment explaining why we are doing this and add a test

Reply:
Done